.github/workflows/build-image.yml

```diff
@@ -18,15 +18,29 @@ on:
       IMAGE_REPO:
         required: true
         type: string
-      TAG:
-        required: true
-        type: string
       BUILD_TYPE:
         required: true
         type: string
 
 jobs:
+  get-prefix:
+    runs-on: ubuntu-latest
+    outputs:
+      FULL_PREFIX: ${{ steps.get-prefix.outputs.FULL_PREFIX }}
+      USED_REPO: ${{ steps.get-prefix.outputs.PREFIX }}
+      TAG_PREFIX: ${{ steps.get-prefix.outputs.TAG_PREFIX }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Compute Prefix
+        id: get-prefix
+        uses: ./.github/actions/compute-prefix
+        with:
+          ORIGINAL_PREFIX: rapidsai/${{ inputs.IMAGE_REPO }}
+          IMAGE_REPOSITORY: ${{ inputs.IMAGE_REPO }}
+          BUILD_TYPE: ${{ inputs.BUILD_TYPE }}
```

Comment:
same message about `get-prefix` here.

if the value is consistent per-workflow, then this should be hoisted to the top-level workflow and passed down to any jobs that need it.